docs/source/working_with_aiida/resultmanager.rst

```diff
@@ -38,7 +38,7 @@ To get all the possible keys that were parsed, you can convert the instance into
 type
 ::
 
-    print list(calc.res)
+    print(list(calc.res))
```

Comment:
There are 2 more instances of incorrect print usages below these. Please fix those as well